cumulus/pallets/parachain-system/proc-macro/src/lib.rs

```diff
@@ -122,8 +122,8 @@ pub fn register_validate_block(input: proc_macro::TokenStream) -> proc_macro::To
 				#[no_mangle]
 				unsafe fn validate_block(arguments: *mut u8, arguments_len: usize) -> u64 {
 					// We convert the `arguments` into a boxed slice and then into `Bytes`.
-					let args = #crate_::validate_block::sp_std::boxed::Box::from_raw(
-						#crate_::validate_block::sp_std::slice::from_raw_parts_mut(
+					let args = #crate_::validate_block::alloc::boxed::Box::from_raw(
+						#crate_::validate_block::alloc::slice::from_raw_parts_mut(
```

Comment:
It is not wrong, but also not particularly useful to re-export `alloc` for a macro.

Reply:
If it is not re-exported here, the other crates would need to write `extern crate alloc` explicitly? Because the crate is not pulled in otherwise.

Reply:
I changed this to re-export only selected types here instead of re-export the whole `alloc` crate